tables/table_json_file.go

```diff
@@ -66,11 +66,21 @@ func listJSONFileWithPath(ctx context.Context, d *plugin.QueryData, h *plugin.Hy
 
 		byteValue, _ := ioutil.ReadAll(jsonFile)
 
+		// Attempt loading as a JSON object by default
 		var result map[string]interface{}
```

Comment:
```suggestion
		var result interface{}
```

@keilin-anz I tried just the suggested change above (without any of your changes below), and it seemed to handle files with JSON arrays OK, and did not produce any breaking changes in how single JSON items were handled either.

Have you tried this approach yet? Can you please give it a try and see if all of the files are decoded in the way you expect?

Reply:
Worked perfectly, and far simpler - thanks!

Looks like this repo uses merge commits, so I've amended my original commit to minimise noise

Just yell out if anything further needed :)